apps/studio/src/server/modules/page/__tests__/page.router.test.ts

```diff
@@ -2015,4 +2016,72 @@ describe("page.router", async () => {
       })
     })
   })
+  describe("scheduled publishing", () => {
```

Comment:
can add 2 basic auth+authorization check?
1. should throw 401 if not logged in
2. should throw 403 if user does not have create access to site

can copy from other test cases - mainly to give more confidence in terms of security

---

can also add another test case for it throwing error if user tries to schedule within 10 minutes

Reply:
PS, can add line break between each section (Arrange, Act, Assert)? for readability purposes. thank you!